### DISCUSSION TITLE
Welcome to spmssite Discussions!

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/spmssite#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wyenelle** [comment]
I'm Favour Timothy 
I was assigned the gallery page, to add carousels for images. I just forked the repo but I can seem to understand the layout of the project 
I need clarity on the structure of the project. I tried adding an HTML file but it wouldn't show on the webpage

**Comment by gbowne1** [comment]
@wyenelle 

    If you forked the repository and opened it in your editor or IDE and it should work especially if you are using Visual Studio Code.  
    
    There is not much structure at the moment.  Just barely got started.  Since this is a new project, I've allowed the person working on the project to style it however they feel is best.  
    
    This project may not be fully suitable for a beginner. 
    
    If you have added an HTML file, you should link it in the index.html where you see this (line 78-92)
    
 ```
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                <a class="dropdown-item" href="./rfq.html">Quotes</a>
                <a class="dropdown-item" href="./capabilities.html">Capabilities</a>
                <a class="dropdown-item" href="./jobs.html">Careers</a>
                <div class="dropdown-divider"></div>
                <a class="dropdown-item" href="#materials">Materials</a>
                <a class="dropdown-item" href="#services">Services</a>
                <a class="dropdown-item" href="#testiominals">Testimonials</a>
                <a class="dropdown-item" href="#equipment">Equipment</a>
                <a class="dropdown-item" href="#industries">Industries</a>
                <div class="dropdown-divider"></div>
                <a class="dropdown-item" href="#history">History</a>
                <a class="dropdown-item" href="#quality">Quality</a>
                <a class="dropdown-item" href="#gallery">Gallery</a>
              </div>

```

You should still be able to see your page.  You can download the HTML file(s) and open it in a browser.
